spec/features/idv/steps/in_person/state_id_controller_spec.rb

```diff
@@ -122,8 +122,7 @@
       )
     end
 
-    context 'same address as id',
```

Comment:
I'd like to chat more about 'allows user to submit valid inputs on form'. (This is around line 45. I am dropping it here so it can be resolved.)
- We are not checking that issuing state and state display on verify info
- If same address as ID, should we be checking count? (ie: address 1 is displayed twice) Maybe too much in the weeds? Maybe enough to check that we made it to the next step and all of the checks on Verify Info are already out of scope

Reply:
This is being tested in the verify info feature specs. Resolving

Reply:
Resolving comment - out of scope for ticket.

Reply:
Good call. I will update the tests to check for issuing state and that is displayed on the verify info page.

Reply:
Observed, resolving.